apps/kiezkasse/phases.py

```diff
@@ -40,5 +40,24 @@ class FeedbackPhase(phases.PhaseContent):
     }
 
 
+class RequestFeedbackPhase(phases.PhaseContent):
+    app = apps.Config.label
+    phase = 'request_feedback'
+    weight = 10
+    view = views.ProposalListView
+
+    name = _('Request and feedback phase')
+    description = _('Get feedback for budgeting requests through rates and '
+                    'comments.')
```

Comment:
in budgeting we use the description: 
`description = _('Request budgeting and get feedback through rates and '
                    'comments.')`
which will be already translated ;)